docs/tutorials/distributed-training.md

```diff
@@ -0,0 +1,39 @@
+# AutoGluon Cloud Distributed Training
+AutoGluon Cloud currently supports distributed training for Tabular.
+
+Tabular predictor trains multiple folds of models underneath and parallelize model training on a single machine. It is natural to expand this strategy to a cluster of machines.
+
+With AutoGluon Cloud, we help you to spin up the cluster, dispatch the jobs, and tear down the cluster. And it is not much different from how you would normally train a `TabularCloudPredictor``. All you need to do is specify `backend="ray_aws"` when you init the predictor
+
+```python
+cloud_predictor = TabularCloudPredictor(
+    ...,
+    backend="ray_aws"
+)
+```
+
+And you would call fit as normal
+
+```python
+cloud_predictor.fit(predictor_init_args=predictor_init_args, predictor_fit_args=predictor_fit_args)
+```
+
+## How to Control Number of Instances in the Cluster
+You can control how many instances are created in the cluster by passing `instance_count` to `fit()`.
+```python
+cloud_predictor.fit(..., instance_count=4)
```

Comment:
If not specified, maybe mention that the way to determine default value will be introduced next?